### ISSUE TITLE
Document API limits so that users know why some information is missing from the API results

### ISSUE THREAD

**Issue by Angelodaniel** [getsentry/sentry#48327]
### Problem Statement

The total number of points Sentry returns is causing some information to be missing from the result of the API call 
https://github.com/getsentry/sentry/pull/31392 that PR was touching this grouping logic last

The most important limit would be this here: https://github.com/getsentry/sentry/blob/89d7aaa5a23f4d4ff962ad12c3be23651ace5c29/src/sentry/snuba/sessions_v2.py#L315
basically its 5_000 devided by the time samples. that means 24 if you query hourly intervals for a day

Depending on the time frame you query for (or rather, the total number of time samples), we limit the number of groups / releases we query for

This is not available as public information

### Product Area

Unknown
